### DISCUSSION TITLE
Having Trouble Building IPA

### DISCUSSION THREAD

**Discussion by munchyk** [qnblackcat/uYouPlus#1264] [Q&A]
Hi!

I might be missing something but from what i understand the only way (as of now) to get an updated IPA version is to build it yourself.

Im receiving the error attached below when trying to build.

Any guidance is greatly appreciated. Tx!

<img width="1859" alt="Screenshot 2023-11-06 at 4 49 51 PM" src="https://github.com/qnblackcat/uYouPlus/assets/55564308/3deecf27-a12d-4bf2-92d3-8d9da61a453b">

**Comment by ashmodai6** [comment]
You can see this error first

==> Downloading uYou…
data.tar.lzma: Truncated ar archive
tar: Error exit delayed from previous errors.
Library/MobileSubstrate/DynamicLibraries/uYou.dylib: Lzma library error: Corrupted input data
tar: Error exit delayed from previous errors.

Seems
https://miro92.com/repo/

Does not work anymore ?

**Reply by munchyk** [reply]
Hey @ashmodai6 ! Thanks for looking into this.

Does that mean it can't be built or will it just require using a different repo?
